airflow/www/views.py

```diff
@@ -2280,10 +2280,13 @@ class VariableView(wwwutils.DataProfilingMixin, AirflowModelView):
     verbose_name_plural = "Variables"
     list_template = 'airflow/variable_list.html'
 
-    def hidden_field_formatter(view, context, model, name):
```

Comment:
It is a hidden field, even if it is invalid. THe reason it can be invalid is that it is a hidden field which needs to be decrypted. Error handling is orthogonal to the fact it is meant to format hidden fields. Anyway, there's no need ot change the name of an existing function.